utils/mqtt_helper.py

```diff
@@ -58,5 +59,7 @@ def send_typed_message(client, topic, message, message_type):
         send_network_chunk_message(client, topic, message)
     elif message_type is MessageType.IMAGE_CHUNK:
         send_image_chunk_message(client, topic, message)
+    elif message_type is MessageType.SIMPLE:
+        send_message(client,topic,message)
```

Comment:
before pushing make sure to do autopep8 --in-place --aggressive --aggressive *.py